### PR TITLE
Add support for rev E HW nxp_fmuk66-e

### PR DESCRIPTION
**Pull request by davids5**
~~This need testing on real HW and a change to Nuttx to be up streamed for the gpiosetevent~~

kinetis gpiosetevent was rejected upstream. 

Testing was done using the Safety switch.

### CODE REVIEW

**Comment by davids5** [comment]
FYI @igalloway , @jarivanewijk

File: boards/nxp/fmuk66-e/init/rc.board_sensors [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+adc start
+
+# Internal Mag I2C bus roll 180, yaw 90
+bmm150 -I -R 10 start
+
+# Onboard I2C baros
+bmp280 -I start
+
+# Internal SPI (accel + mag)
```
Comment (bkueng) [inline boards/nxp/fmuk66-e/init/rc.board_sensors:14]:
```suggestion
# Internal SPI (accel + gyro)
```

**Comment by davids5** [comment]
Rebased on master

**Comment by davids5** [comment]
@dagar - would you please merge when CI is done?

**Pull request merged**